app/models/pawn.rb

```diff
@@ -1,6 +1,29 @@
 class Pawn < Piece
-  def move_valid?
-    # add logic here
+  def move_valid?(x, y)
+    return false if space_occupied?(x, y) == true
+    return false if legal_move?(x, y) == false
```

Comment:
Actually, legal_move? is going to be calling other things later (diagonal capture, en passant capture).